src/vsr/replica.zig

```diff
@@ -5336,24 +5510,27 @@ pub fn ReplicaType(
                 }
 
                 self.ping_timeout.start();
-                self.commit_timeout.stop();
-                self.normal_status_timeout.start();
+                self.commit_message_timeout.stop();
+                self.normal_heartbeat_timeout.start();
+                self.start_view_change_window_timeout.start();
+                self.start_view_change_message_timeout.start();
                 self.view_change_status_timeout.stop();
                 self.view_change_message_timeout.stop();
                 self.repair_timeout.start();
             }
 
+            self.heartbeat_timestamp = 0;
```

Comment:
Reset any time the primary changes since different replicas have different clocks.